projects/RabbitMQ.Client/Impl/RabbitMQActivitySource.cs

```diff
@@ -56,6 +67,20 @@ public static class RabbitMQActivitySource
             new KeyValuePair<string, object?>(ProtocolVersion, "0.9.1")
         };
 
+        internal static Activity? OpenConnection(IFrameHandler frameHandler)
+        {
+            if (!s_connectionSource.HasListeners())
+            {
+                return null;
+            }
+
+            Activity? connectionActivity =
+                s_connectionSource.StartRabbitMQActivity("rabbitmq connect", ActivityKind.Client);
```

Comment:
Please rename "rabbitmq connect" to "connection.open". It is obvious that this client will connect to a RabbitMQ node.

Reply:
ok to rename but activity name is used in all UI when traces are ingested, I highly recommand to set a more human friendly name like "connection open"

Reply:
"connection open" (with a space) is not grammatically correct. "connection.open" is a protocol frame name but I can see how that'd be too obscure.

"connection attempt" can be used before we know that the connection succeeded. "connection opened" can be used after a successful connection. In fact, both can be used at the same time in different places.

Reply:
"connection attempt"  should be good because we do not know if its a success. 
Span status will be set to ERROR and an exception event will be attached to the current span if it fails.